.styleci.yml

```diff
@@ -1,4 +1,6 @@
-preset: psr2
+preset: recommended
 
-enabled:
-  - return
+disabled:
+  - align_double_arrow
+  - concat_without_spaces
```

Comment:
Do you want to also turn on `concat_with_spaces`?

Reply:
yes let's do that!